.nuspec/Xamarin.Forms.nuspec

```diff
@@ -34,8 +34,10 @@
       <group targetFramework="tizen40">
         <dependency id="Tizen.NET" version="4.0.0"/>
       </group>
+      <group targetFramework="Xamarin.iOS10">
+        <dependency id="Xamarin.iOS.MaterialComponents" version="60.1.0"/>
```

Comment:
We want to avoid this long term, correct?  Forms Core should not need this dep.

Reply:
I don't think we are avoiding it.  It's required for the Material Visual components on iOS and the only way to not have the dep would be to create a separate Xamarin.Forms.Material package or just not use material components on iOS.

Reply:
I would like to find a solution where this dep is optional and users that use Material will need to add this dep explicitly.  Need to think about the user experience. 

At the very least we will need to ensure that it does link out when no in use.

Reply:
I think it won't be easy, if the dll was built using this there's no way they can build after instal the Nuget without installing this irc. 
Only way is we ship the dll and everything needed on our package. 

But in the UX field, users will just update a package that will install this package, there will be more stuff there but the update experience should just work. and Linker should do the rest of keeping the app small.